app/models/foreman_azure_rm/azure_rm.rb

```diff
@@ -243,7 +244,7 @@ def create_vm(args = {})
       logger.debug "Virtual Machine #{args[:vm_name]} Created Successfully."
       create_vm_extension(region, args)
       # return the vm object using azure_vm
-      AzureRmCompute.new(azure_vm: vm, sdk: sdk, resource_group: args[:resource_group], nics: vm_nics(vm))
+      AzureRmCompute.new(azure_vm: vm, sdk: sdk, resource_group: args[:resource_group], nics: vm_nics(vm), script_command: user_command, script_uris: args[:script_uris])
```

Comment:
@ShimShtein for `create_vm`, this still doesn't show the values on edit page.

Reply:
This is because we don't set the `script_command` in `new_vm` method.
We need to set the attribute in all calls to `AzureRmCompute.new`

Reply:
[comment above in new_vm](https://github.com/theforeman/foreman_azure_rm/pull/63/files#r365808899).
Also, it works right for new_vm (compute profile) while not just for create_vm.